var/spack/repos/builtin/packages/apex/package.py

```diff
@@ -135,6 +138,11 @@ class Apex(CMakePackage):
     # https://github.com/UO-OACISS/apex/issues/180.
     conflicts("~kokkos", when="@:2.6.5")
 
+    # OpenCL added with 2.7.0
+    conflicts("+opencl", when="@:2.6.5")
```

Comment:
This was the conflict I was referring to with the previous comment on `variant("opencl", ...)`. I think it can be removed?

Reply:
yes, it can